kubespawner/objects.py

```diff
@@ -110,13 +113,6 @@ def make_pod_spec(
 
     pod.spec = V1PodSpec()
 
-    security_context = V1PodSecurityContext()
```

Comment:
Can you leave this here too? No harm in having them both, and I'd rather err on the side of overspecifying than underspecifying in this context.